src/components/admin/Select/Select.tsx

```diff
@@ -0,0 +1,54 @@
+import { FieldError, UseFormRegister } from "react-hook-form";
+
+import { AnyForm } from "types/utility";
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+
+export interface Option {
+  name: string;
+  value: string;
+}
+
+type SelectProps = {
+  options: WithId<AnyVenue, string>[];
+  register: UseFormRegister<AnyForm>;
+  name: string;
+  error?: FieldError;
+};
+
+export const Select: React.FC<SelectProps> = ({
+  options,
+  register,
+  name,
+  error,
+}) => {
+  return (
+    <div className="relative inline-block w-full text-gray-700">
+      <select
+        className="w-full h-10 pl-3 pr-6 text-base placeholder-gray-600 border rounded-lg appearance-none focus:shadow-outline"
+        placeholder="Regular input"
+        {...register(name)}
+      >
+        {options?.map((option) => (
+          <option
+            key={option.name}
+            className="bg-white w-full h-10 pl-3 pr-6 text-base placeholder-gray-600 border rounded-lg appearance-none focus:shadow-outline"
```

Comment:
These could be moved  to `Select.tailwind.ts` so the code is more concise in here